source/administration/config-in-database.rst

```diff
@@ -0,0 +1,32 @@
+Configuration in the Mattermost Database
+=========================================
+A new configuration option was added in `5.10 <https://docs.mattermost.com/administration/changelog.html#configuration-in-database>`_ to use the database for the single source of truth of the active configuration of your Mattermost installation. This will change the Mattermost binary from reading the default config.json file to reading the configuration settings stored within a configuration table in the database. Mattermost has been running our high availability community server on this option since the feature was released.  
+
+Benefits to using this option include: 
+
+  - Conveniently manage configuration changes directly from the System Console.
+  - In high availability deployments, configuration changes are applied to all nodes instantaneously as possible removing the need to update individual configuration files on each server and ensures all servers have the correct configurations in the event  a server in the node goes down.
+  - Also in high availability deployments, the read-only state of the System Console is removed.   
+
+To change your server from reading the config.json file to using the configuration in database: 
```

Comment:
Can't make these suggestions all in place, but propose replacing as follows:

```
To start using configuration in database, pass the database connection string via the `--config` flag or `MM_CONFIG` environment variable. For example:

    ```
    ./mattermost --config="postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable\u0026connect_timeout=10"
    ```

To migrate an existing config.json into the database, use the ``config migrate`` `command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-config-migrate>`_. For example:

    ```
    ./mattermost config migrate  path/to/config.json "postgres://mmuser:mostest@dockerhost:5432/mattermost_test?sslmode=disable&connect_timeout=10"
    ```

Any existing SAML certificates and private keys will also be migrated to the database.
```